src/pages/Login/Login.scss

```
@@ -0,0 +1,223 @@
+        cursor: pointer;
+      }
+    }
+
+    .column-content {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+
+      .hero-wrapper {
+        .title {
+          margin-bottom: 16px;
+          color: $ikea-yellow;
+          font-size: 32px;
+          font-weight: 700;
+        }
+
+        p {
+          padding: 12px 0;
+          color: #ffffff;
+          font-size: 14px;
+          line-height: 1.3;
+        }
+      }
+
+      .kiosk-label {
+        padding: 2px 0;
+        color: #ffffff;
+        font-size: 11px;
+      }
+    }
+  }
+
+  .right {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    padding: 40px 80px;
+
+    .right-content-container {
+      width: 100%;
+      max-width: 432px;
+    }
+
+    .feature-otp {
+      display: flex;
+      align-items: center;
+      color: #484848;
+      font-size: 14px;
+
+      .underline {
+        font-weight: 700;
+        text-decoration: underline;
+        cursor: pointer;
+      }
+    }
+
+    .warning {
+      display: flex;
+      align-items: center;
+      margin-top: 8px;
+      margin-right: 4px;
+      color: #ed4407;
+      font-size: 14px;
+
+      .icon {
+        width: 16px;
+        height: 16px;
+      }
+    }
+
+    label {
+      display: block;
+      margin-top: 36px;
+      margin-bottom: 4px;
+      font-size: 14px;
+    }
+
+    input {
+      display: block;
+      width: 100%;
+
+      height: 48px;
+      margin-top: 8px;
+      padding: 0;
+      padding-inline-start: 4px;
+      border: 1px solid #656565;
+      border-radius: 5px;
+
+      &:focus {
+        border-color: #325eed;
+      }
+    }
+
+    input[type='text'] {
+      font-size: 16px;
+    }
+
+    input[type='password'] {
+      font-size: 16px;
+    }
+
+    .reveal-container {
+      display: flex;
+      align-items: center;
+      color: $ikea-blue;
+      font-size: 14px;
+      cursor: pointer;
+      user-select: none;
+
+      p {
+        margin-left: 4px;
+      }
+
+      &:hover {
+        color: #032aa9;
+      }
+    }
+
+    .checkbox-container {
+      display: flex;
+      align-items: center;
+      margin-top: 24px;
+
+      .checkbox {
+        position: absolute;
+        width: 22px;
+        height: 22px;
+        margin: 0;
+        padding: 0;
+      }
+
+      .label {
+        margin-left: 40px;
+        color: #484848;
+        font-size: 14px;
+        line-height: 1.3;
+        word-break: keep-all;
+        white-space: pre-wrap;
+      }
+    }
+
+    .submit-btn {
+      width: 100%;
+      margin-top: 32px;
+      padding: 18px 0;
+      transition: 0.3s;
+      border: 0;
+      border-radius: 64px;
+      background-color: $ikea-blue;
+      color: #ffffff;
+      font-size: 14px;
+      font-weight: 700;
+      cursor: pointer;
+
+      &:hover {
+        background-color: #055093;
+      }
+    }
+
+    .submit-btn.disabled {
+      opacity: 0.3;
+    }
```

```suggestion
    .disabled {
      opacity: 0.3;
    }
```

위의 리뷰와 연관 지으면 이렇게 수정할 수 있겠습니다.